### ISSUE TITLE
Draggable not working with group points in "column" chart 

### ISSUE THREAD

**Issue by se-panfilov** [highcharts/highcharts#11337]
#### Expected behaviour
I have grouped points by `groupId`. I expect grouped points to move along.
It does work for a `line` chart.
But it does no effect for a `column` chart type - all the points still drag independently.

#### Actual behaviour
`column` chart doesn't respect `groupBy` property. 

#### Live demo with steps to reproduce
https://jsfiddle.net/se_panfilov/0pbhvs36/6/

#### Product version
Latest

#### Affected browser(s)
Chrome latest

**Comment by pawelfus** [comment]
Thanks for reporting the issue @se-panfilov 

It seems to work as designed - drag&drop for points works when `groupBy` option is set. When **resizing** a column only current point is taken into account. When dragging points, we get action for all points: https://jsfiddle.net/BlackLabel/hmrn3t5u/1/

With a simple workaround (`Highcharts.seriesTypes.column.prototype.dragDropProps.y.move = true;`), we can this effect: http://jsfiddle.net/BlackLabel/mprL7jzw/10/ - drag&drop for the control point changes y-value for the column, drag&drop on the rectangle changes values for all grouped points.

@oysteinmoseng - Can we should make this configurable?

**Comment by oysteinmoseng** [comment]
@pawelfus Yes, I think it is a great enhancement, but ideally the enhancement should be to make groupBy resize configurable. While the workaround above works, it only works if you move the point without using the drag handle - which is a bit unintuitive. Ideally it should be configurable to make the resizing use grouping as well (for all series).

It should be possible to create a resizing workaround using custom drag events: https://api.highcharts.com/highcharts/series.column.point.events.drag

**Comment by sebastianbochan** [comment]
Unstale.

Reopen because of enhancement.
